tests/conftest.py

```diff
@@ -72,3 +73,11 @@ def mod_device() -> Device:
             ),
         ),
     )
+
+
+@pytest.fixture()
+def patch_plt_show(monkeypatch):
+    # Close residual figures
+    plt.close("all")
+    # Closes a figure instead of showing it
+    monkeypatch.setattr(plt, "show", plt.close)
```

Comment:
So we no longer test the diplaying of the plot ? As far I have understood, monkey patching is different as patching https://stackoverflow.com/questions/5626193/what-is-monkey-patching

Reply:
We never did test it, because we never called `plt.show()`. The only difference to what I'm doing here is that I'm taking the opportunity to close the figure, instead of just suppressing the call.